sktime/utils/parallel.py

```diff
@@ -0,0 +1,22 @@
+from joblib import Parallel
+
+def generate_dataset_batches(X, parallel):
```

Comment:
what is this for? some short docstring description would be helpful

Reply:
Legacy parallel code, will delete it